icu4c/source/data/unit/de.txt

```diff
@@ -10,12 +10,42 @@ de{
     units{
         acceleration{
             g-force{
+                case{
+                    accusative{
+                        one{"{0} g-Kraft"}
+                        other{"{0} g-Kraft"}
+                    }
+                    dative{
```

Comment:
This is really a comment for the CLDR structure & data, but seems like for situations like this where there are no distinctions in value for some or al of the cases, to reduce data size we might want to have a case "other" or "fallback" that can provide the default value if the specific case is not listed.

Reply:
The current "other" or "fallback" is the _parent_ of "case" - that's the "nominative" case.

My current proposal being this, keeping existing structure as much the same as possible:

            g-force{
                case{
                    accusative{
                        one{"{0} g-Kraft"}
                        other{"{0} g-Kraft"}
                    }
                    dative{
                        one{"{0} g-Kraft"}
                        other{"{0} g-Kraft"}
                    }
                    genitive{
                        one{"{0} g-Kraft"}
                        other{"{0} g-Kraft"}
                    }
                }
                dnam{"g-Kraft"}
                gender{"feminine"}
                one{"{0} g-Kraft"}
                other{"{0} g-Kraft"}
            }

A cleaner suggestion would be this, when I call it **other** instead of **nominative** - since I'm also not sure whether it's linguistically "correct" to call it nominative for languages that don't have case? ;-) :

            g-force{
                case{
                    other{
                        one{"{0} g-Kraft"}
                        other{"{0} g-Kraft"}
                    }
                    accusative{
                        one{"{0} g-Kraft"}
                        other{"{0} g-Kraft"}
                    }
                    dative{
                        one{"{0} g-Kraft"}
                        other{"{0} g-Kraft"}
                    }
                    genitive{
                        one{"{0} g-Kraft"}
                        other{"{0} g-Kraft"}
                    }
                }
                dnam{"g-Kraft"}
                gender{"feminine"}
            }

Reply:
Spec formaly falls back to "nominative", then to "no case":
https://www.unicode.org/reports/tr35/tr35.html#Lateral_Inheritance

I suggest that's very much an argument in favour of calling it "nominative", not "other", if I go with moving nominative into the "case" table.